docs/configuration/alertmanager-integration/grafana-alert-manager.rst

```diff
@@ -1,25 +1,98 @@
 Grafana AlertManager
 ****************************************
 
-Grafana can send alerts to Robusta for automatic enrichment and visualization.
+Grafana can send alerts to the Robusta timeline for visualization and AI investigation.
 
 .. image:: /images/grafana-docs-robusta-ui.png
   :width: 600
   :align: center
 
 
-This guide only covers integrating alerts from Grafana Alerting with Robusta, not configuring Robusta to query metrics from the relevant Grafana data source.
+This guide only covers integrating alerts from Grafana Alerting with Robusta.
 
-After completing this tutorial, we recommend configuring a metrics integration according to the :ref:`standard instructions for your metrics backend <Integrating with Prometheus>`
-
-Prerequisite
+Kubernetes Alerts
```

Comment:
I would put this section beneath Send Alerts to Robusta's timeline. It only makes sense after you've covered the basics.

Reply:
fixed